src/main/java/org/fogbowcloud/manager/core/models/orders/ComputeOrder.java

```diff
@@ -115,4 +121,8 @@ public String getPublicKey() {
         return publicKey;
     }
 
+    public List<String> getNetworksId() {
+        return networksId;
```

Comment:
I think that would be better to return a Collections.unmodifiableList(networksId), and have a set method for networksId, instead of returning the list itself.